test_scripts/RC/InteriorVehicleData_cache/025_GetInteriorVD_limitation_rejected.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0178-GetInteriorVehicleData.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case
+-- 1. In .ini file GetInteriorVehicleDataRequest = 10,20
+-- 2. Mobile app sends 11 GetInteriorVD(module_1, without subscribe parameter) requests per 20 sec
+-- SDL must
+-- 1. reject 11 GetInteriorVD(module_1, without subscribe parameter) request with result code REJECTED
```

Comment:
@GetmanetsIrina Please correct `11` -> `11th`. Because it looks like all 11 has to be rejected.